rmk-macro/src/bind_interrupt.rs

```diff
@@ -93,6 +101,7 @@ pub(crate) fn bind_interrupt_default(keyboard_config: &KeyboardTomlConfig) -> To
                     .support_le_2m_phy()?
                     .peripheral_count(1)?
                     .buffer_cfg(L2CAP_MTU as u16, L2CAP_MTU as u16, L2CAP_TXQ, L2CAP_RXQ)?
+                    #tx_power
```

Comment:
Please move `#tx_power` before the `peripheral_count`, to ensure that it can be set correctly